hybrid/pv_source.py

```diff
@@ -69,6 +69,7 @@ def system_capacity_kw(self, size_kw: float):
         :return:
         """
         self._system_model.SystemDesign.system_capacity = size_kw
+        self._financial_model.value('system_capacity', size_kw) # needed for custom financial models
```

Comment:
Can we assume variables in the system and financial models with the same name be linked?

Also, what if `Singleowner` is user-instantiated, thus its parameters are not linked with the system model? How do we ensure they are updated before the model executes? Since `Singleowner` is built-in, we cannot overload `execute` or add `set_financial_inputs` to it like we are for the custom financial models.

Reply:
I think if they have the same name, they should be linked, either explicitly or implicitly. The way this is done here, you'll have to add this line to all the `x_source.py` files